app/src/main/java/com/mobileproto/hireddit/hireddit/MainActivity.java

```diff
@@ -11,6 +12,7 @@
 
 public class MainActivity extends AppCompatActivity
```

Comment:
NORA IS COMMENTING : shouldn't this be in the fragment? since we don't really use the main activity for view stuff and everything?